packages/eslint-plugin/eslint.config.mjs

```diff
@@ -50,7 +50,7 @@ const config = [
             "jsonc/sort-keys": "error",
         },
     },
-    packageJson,
+    packageJson.configs.recommended,
```

Comment:
This is done as described here: https://www.npmjs.com/package/eslint-plugin-package-json#usage